pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java

```diff
@@ -49,70 +50,259 @@
+            name = "authentication",
+            value = "Class name od authentication plugin of the client."
+    )
     private String authPluginClassName;
 
+    @ApiModelProperty(
+            name = "authParams",
+            value = "Authentication params of the client."
+    )
     @Secret
     private String authParams;
+
+    @ApiModelProperty(
+            name = "authParamMap",
+            value = "Authentication map of the client."
+    )
     @Secret
     private Map<String, String> authParamMap;
 
+    @ApiModelProperty(
+            name = "operationTimeoutMs",
+            value = "Client operation timeout, the unit is milliseconds."
+    )
     private long operationTimeoutMs = 30000;
+
+    @ApiModelProperty(
+            name = "statsIntervalSeconds",
+            value = "The Interval to print client stats, the unit is seconds."
+    )
     private long statsIntervalSeconds = 60;
 
+    @ApiModelProperty(
+            name = "numIoThreads",
+            value = "Number of IO threads."
+    )
     private int numIoThreads = 1;
+
+    @ApiModelProperty(
+            name = "numListenerThreads",
+            value = "Number of consumer listener threads."
+    )
     private int numListenerThreads = 1;
+
+    @ApiModelProperty(
+            name = "connectionsPerBroker",
+            value = "The number of connections established between the client and each Broker."
+    )
     private int connectionsPerBroker = 1;
 
+    @ApiModelProperty(
+            name = "useTcpNoDelay",
+            value = "Whether to use Tcp NoDelay option."
+    )
     private boolean useTcpNoDelay = true;
 
+    @ApiModelProperty(
+            name = "useTls",
+            value = "Whether to use TLS."
+    )
     private boolean useTls = false;
+
+    @ApiModelProperty(
+            name = "tlsTrustCertsFilePath",
+            value = "Path to the trusted TLS certificate file."
+    )
     private String tlsTrustCertsFilePath = "";
+
+    @ApiModelProperty(
+            name = "tlsAllowInsecureConnection",
+            value = "Whether the client accepts untrusted TLS certificates from the broker."
+    )
     private boolean tlsAllowInsecureConnection = false;
+
+    @ApiModelProperty(
+            name = "tlsAllowInsecureConnection",
+            value = "Whether the hostname is validated when the proxy creates a TLS connection with brokers."
+    )
     private boolean tlsHostnameVerificationEnable = false;
+    @ApiModelProperty(
+            name = "concurrentLookupRequest",
+            value = "The number of concurrent lookup requests that can be sent on each broker connection. "
+                    + "Setting a maximum helps to keep from overloading brokers."
+    )
     private int concurrentLookupRequest = 5000;
+
+    @ApiModelProperty(
+            name = "maxLookupRequest",
+            value = "The maximum number of lookup requests allowed on "
+                    + "each broker connection to prevent overload on broker."
```

Comment:
```suggestion
            value = "Maximum number of lookup requests allowed on "
                    + "each broker connection to prevent overloading a broker."
```